strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/cron/jobs/RebuildMavenMetadataCronJobTestIT.java

```diff
@@ -132,30 +119,44 @@ public void testRebuildArtifactsMetadata(TestInfo testInfo)
                          jobName,
                          RebuildMavenMetadataCronJob.class,
                          STORAGE0,
-                         repositoryId,
+                         RMMCJTIT_SNAPSHOTS,
                          properties -> properties.put("basePath", "org/carlspring/strongbox/strongbox-metadata-one"));
 
         await().atMost(EVENT_TIMEOUT_SECONDS, TimeUnit.SECONDS).untilTrue(receivedExpectedEvent());
     }
 
+    @ExtendWith({ RepositoryManagementTestExecutionListener.class, ArtifactManagementTestExecutionListener.class })
     @Test
-    public void testRebuildMetadataInRepository()
+    public void testRebuildMetadataInRepository(@MavenRepository(storageId = STORAGE0,
+                                                                 repositoryId = TRMIR_SNAPSHOTS)Repository repository,
+                                                @MavenTestArtifact(repositoryId = TRMIR_SNAPSHOTS,
+                                                                   id = "org.carlspring.strongbox:strongbox-metadata-one",
+                                                                   versions = {"2.0-20190512.202015-1",
+                                                                               "2.0-20190512.202015-2",
+                                                                               "2.0-20190512.202015-3",
+                                                                               "2.0-20190512.202015-4",
+                                                                               "2.0-20190512.202015-5"},
+                                                                   classifiers = {"javadoc",
+                                                                                  "sources",
+                                                                                  "source-release"})List<Path> artifact1,
+                                                @MavenTestArtifact(repositoryId = TRMIR_SNAPSHOTS,
+                                                        id = "org.carlspring.strongbox:strongbox-metadata-second",
+                                                        versions = {"2.0-20190512.202015-1",
+                                                                "2.0-20190512.202015-2",
+                                                                "2.0-20190512.202015-3",
+                                                                "2.0-20190512.202015-4",
+                                                                "2.0-20190512.202015-5"},
+                                                        classifiers = {"javadoc",
+                                                                "sources",
+                                                                "source-release"})List<Path> artifact2)
```

Comment:
Cosmetic: Could you please follow our coding convention (as above)? Thanks! :)

Reply:
This is done. Thanks!